tests/integration/test_integration_all_rust.py

```diff
@@ -1448,9 +1457,35 @@ async def test_broadcast_update_on_connect(self):
         MOCK_MP_SERVICES = {"kinto:123": "ver1"}
         MOCK_MP_POLLED.clear()
         MOCK_MP_POLLED.wait(timeout=5)
+        log.critical("MEGA")
```

Comment:
Why are you logging these as `critical`?

Reply:
For the bold red to make it easy to find the debug messages. All those log and err messages are removed now.  Turned out the issue was not with the tests, but autopush throwing an error when the `v1/broadcasts` did not return a response in the time it was setup.

Reply:
Ah, heh. Sorry. 

FWIW, Lina introduced me to the idea of using leading emoji for debugging lines. Turns out many folk process the colors and shapes faster than text allowing you to quickly skim complex logs for distinct patterns. (It works best for displays that can support emoji fonts, so VT100 style will probably not cut it.)

Reply:
Great idea, thx for the suggestion, I have seen it in the repo and it's nice.